app/models/site.rb

```diff
@@ -228,6 +228,12 @@ def multisearch(*args)
     PgSearch.multisearch(*args).where(site: self)
   end
 
+  def ilike_search(*args)
```

Comment:
But this method is not used anymore, rigth?

Reply:
Ups, yes, it has to be removed